Changelog

```diff
@@ -1,3 +1,45 @@
+    - FEATURE: Add `Copy Comment` to the torrent list's context menu (thalieht) 30d9978c97a582b396c53441c938b05b124ba8c9
+    - FEATURE: Allow relative profile paths (Victor Chernyakin) 0297f0f34baa51cde6087edd2f521f108109ad2e
+    - FEATURE: Enable Ctrl+F hotkey for more inputs (thalieht) 8dcc734df562885131dbd0c2cc4ec31e24361559
+    - FEATURE: Add seeding limits to RSS and Watched folders options UI (glassez) d652a10495fdd5175242f029a7e2b9a6373ea985
+    - BUGFIX: Display error message when unrecoverable error occurred (glassez) a0e41a11dec9e195455d990fd0e87390882c703d
+    - BUGFIX: Update size of selected files when selection is changed (glassez) eddfaedc4d34cfcaf8a38210e6700ace0886cc9f
+    - BUGFIX: Specify a locale if none is set (Chocobo1) fc56fdade75610fabf42426cdc8960b9f1ba77a4
+    - WEBUI: Improve WebUI responsiveness (Chocobo1) 2edb1a0765dc501bb66d133d1bc8ac3c37704f51
+    - WEBUI: Add I2P settings to WebUI (thalieht) a3888811f6ad5a82119805378730a68402765bfa
+    - WEBUI: Do not exit the app when WebUI hass failed to start (Hanabishi) 35e4b35d125983051a25173d595119d99b7bda3a
+    - WEBUI: Add `Moving` filter to side panel (xavier2k6) f067ab169224013cb4bf21630b8a3eaa5d7d2dd6
+    - WEBUI: Add dark theme (d47081) 9d90141c29b80b03631e19cba0fa6f1cddc30423
+    - WEBUI: Allow to remember torrent content files deletion (David) e69f85782881a4b831ed97694263f027ecfa568d
+    - WEBUI: Leave the fields empty when value is invalid (Chocobo1) 54bcabeca4292a72649202a03eaf09e711d73395
+    - WEBUI: Use natural sorting (Chocobo1) 00372dd559dedd19e3bcb66db8e7d5faf597d4b2
+    - WEBUI: WebAPI: Get torrent count  (glassez) cecbb44a031c96e8fce48f12eeddda47b9c85a27
+    - WEBUI: WebAPI: Fix wrong timestamp values  (Chocobo1) 114652205c92bf7fcf63f355cab439a5eb4b184f
+    - WEBUI: WebAPI: Allow WebAPI to specify filename and mime type for result data  (glassez) 60bb819e2e839d691dec6915d592d4a2c66d1de2
+    - RSS: Show RSS feed title in HTML browser (Jay) f3b688217eb518ca67de23edd716278efde0b990
+    - SEARCH: Allow users to specify Python executable path (Chocobo1) b3fda76027859707b57530a1de0e2c68c0b27d9f
+    - WINDOWS: Use Fusion style on Windows 10+. It has better compatibility with dark mode (glassez) 5a334175a8b5047cf56552c0bf5a54c3e3e14893
+    - WINDOWS: Improve .torrent association handling (Chocobo1) cba433823ff46c5c799e8d9e7c6dd65c9e8bdd4c
+    - WINDOWS: Allow to set qBittorrent as default program (glassez) 6860c0d60de0ad2bb4374096354fadf3d0261a43
+    - LINUX: Add support for systemd power management (Chocobo1) f3f9cfe44ed49d5bf3de0e318fa9c8ec48645ca4
+    - LINUX: Support for localized man pages (Victor Chernyakin) 2dc1a7d66f246faee94f08c1a033f107ae126f0f
+    - MACOS: Display download/upload speed in dock icon (Nick Korotysh) e6ec3d0c2bbdda2d3228558907353f725a4d45b7
+    - MACOS: Add support for quarantine on macOS (Chocobo1) 98576dacae1ea6fb5369f4aef2cf91426766cb57
+    - OTHER: Switch to C++20
```

Comment:
I believe this and the following one correlate with `FEATURE: Drop support for Qt5, qmake, autotools, Windows < 10, Windows 32-bit`.

Reply:
IMO, I prefer to be a tad bit explicit on what is dropped.

Reply:
My comment is mostly about different categories used for similar kind of changes.